spec/controllers/bids_controller_spec.rb

```diff
@@ -67,6 +69,39 @@
     end
   end
 
+  describe '#index' do
+    let(:bids) {
+      [3400, 3000, 1800].map do |amount|
+        Bid.create({
+          auction_id: auction.id,
+          amount: amount,
+          bidder_id: current_bidder.id
+        })
+      end
+    }
+    context 'when logged in' do
+      before do
+        allow(controller).to receive(:current_user).and_return(current_bidder)
+      end
+
+      it 'renders the template' do
+        get :index, auction_id: auction.id
+        expect(response).to render_template(:index)
+      end
+    end
+
+    context 'when logged out' do
```

Comment:
so people can see this view without being logged in? If that is the case, I think it would be helpful to say what http status code is returned for logged out users than to have a test for which template is returned